zerver/lib/bugdown/__init__.py

```diff
@@ -1200,6 +1201,17 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in realm_slash_commands:
+                escaped = re.escape(("/" + command).lower())
```

Comment:
I agree; the standard for this feature from other tools is that slash commands MUST be at the start of a message.